### PR TITLE
DOCS: Add link to new Cluster Recovery gpcr utility to GPDB utility reference

### PR DESCRIPTION
**Pull request by VMWareJennyBeth**

### CODE REVIEW

File: gpdb-doc/markdown/utility_guide/utility-programs.html.md [hunk 1]
```diff
@@ -41,6 +41,7 @@ Greenplum Database provides the following utility programs. Superscripts identif
 - [gpmovemirrors](ref/gpmovemirrors.html)
 - [gpmt](ref/gpmt.html)
 - [gppkg](ref/gppkg.html)
+- [gprc](https://docs.vmware.com/en/VMware-Greenplum-Cluster-Recovery/1.0/vmware-greenplum-cluster-recovery/GUID-ref-gpcr.html)
```
Comment (jimmyyih) [inline gpdb-doc/markdown/utility_guide/utility-programs.html.md:44]: Should be `[gpcr]`.

Comment (VMWareJennyBeth) [reply gpdb-doc/markdown/utility_guide/utility-programs.html.md:44]: THANK YOU for catching this crucial typo, @jimmyyih !!!

**Review by jimmyyih — APPROVED** [review]
The link leads to a 404 but I'm assuming it'll be a valid link later.

**Comment by VMWareJennyBeth** [comment]
> 

Correct, it will be a live link once Cluster Recovery Beta docs go live.

**Pull request merged**